### PR TITLE
File > New Data Frame

### LINKED ISSUE #5097: Adding Try to the File > New dialogue
The new File > New Data frame is working very well.  I am delighted!

a) Could the **Try** control be added?  It is in the **Prepare > Calculate > Calculations**, and **Prepare > Generate > Enter**.  It would be good to have it, in each of our "half-way" dialogues.
b) Now wakefield is added automatically when R-Instat is loaded I can just have the command r_data(500,the_works) and I get an interesting data set with 49 columns.  This could be added as one the built-in examples perhaps.  (While we await the random option in the dialogue.
We might want to give it as wakefield::r_data(500,"the_works") just to emphasise the package, though it isn't necessary.
c) Would it make sense for some right-click operations to be included - a subset of what is in the script window?  An example I have now would benefit (in the command option) if the first line could just be run separately.  Then the remainder would be run with the last line (as usual) making the data frame.  I can do this now by running the first line from the script window. , but a bit more convenient to simply run from here.

### PR DESCRIPTION
**Pull request by Patowhiz**
Fixes #5097
@africanmathsinitiative/developers this is ready for review.

### CODE REVIEW

File: instat/dlgNewDataFrame.vb [hunk 1]
```diff
@@ -296,4 +297,31 @@ Public Class dlgNewDataFrame
...
+            If ucrInputCommand.IsEmpty Then
+                ucrInputTryMessage.SetText("")
+                ucrInputTryMessage.txtInput.BackColor = Color.White
+            Else
+                strVecOutput = frmMain.clsRLink.RunInternalScriptGetOutput(ucrInputCommand.GetText, bSilent:=True)
+                If strVecOutput IsNot Nothing Then
+                    'If strVecOutput.Length > 0 Then
+                    'End If
+                    ucrInputTryMessage.SetText("Command Ok.")
+                    ucrInputTryMessage.txtInput.BackColor = Color.White
```
Comment (Patowhiz) [inline instat/dlgNewDataFrame.vb]: @dannyparsons though not urgent, we will have add some extra methods "features" into our custom controls. Looking at the code for most dialogues, I'm concerned about how we have to access the inner controls of the custom controls, this violates OOP principle (Encapsulation).

Comment (dannyparsons) [reply instat/dlgNewDataFrame.vb]: Yes agreed, and happy for you to add an issue on that with the methods to be done, or just to do it yourself in a separate PR.

**Comment by maxwellfundi** [comment]
@Patowhiz  any progress on this?

**Comment by Patowhiz** [comment]
@maxwellfundi this is indicated as ready for review above

**Comment by maxwellfundi** [comment]
@dannyparsons have a look at this too.

**Comment by dannyparsons** [comment]
I can have a look but someone needs to review as well.

**Comment by stevenndungu** [comment]
Reviewed. It works as expected.
@Patowhiz Should Ok be enabled iff the try solves to "command ok" or?

**Comment by stevenndungu** [comment]
Noticed similar characteristics in dlg Enter and Calculations, @dannyparsons this is ready for you to review.

**Comment by Patowhiz** [comment]
@stevenndungu not really, the user can actually just go directly to click **ok** even without trying the command first.

**Comment by stevenndungu** [comment]
Yes for sure. I understand why it is left like that. Thanks.

**Pull request merged**